src/launcher/components/AppManagementView.jsx

```diff
@@ -10,6 +10,7 @@ import { bool, func, instanceOf, string } from 'prop-types';
 
 import AppManagementFilter from '../containers/AppManagementFilterContainer';
 import ReleaseNotesDialog from '../containers/ReleaseNotesDialogContainer';
+import ContainerWithScrollbar from '../containers/WithScrollbarContainer';
```

Comment:
I'd suggest either using a named export, or call the defaulted exported the same everywhere it's used, since having different names will make it trickier to find usages in the future.

So here we would call it `WithScrollbarContainer` instead of `ContainerWithScrollbar` (or change it the other way around).